physionet-django/console/forms.py

```diff
@@ -975,7 +973,14 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.fields['access_policy'].choices = AccessPolicy.choices(gte_value=AccessPolicy.RESTRICTED)
+        self.fields['access_policy'].choices = AccessPolicy.choices(gte_value=AccessPolicy.CREDENTIALED)
```

Comment:
This looks like it is undoing https://github.com/MIT-LCP/physionet-build/pull/1519/commits/7a12b0d12d36b86607efb9748db5cc9608089c10.

It should be possible to assign a DUA to restricted access projects, so should this stay as `gte_value=AccessPolicy.RESTRICTED`?

Reply:
Yeah, good catch. I don't know which commit reverted this change - I'll make it correct again and let you know.

Reply:
Brought `AccessPolicy.RESTRICTED` back.